src/app/components/tag/TagInput.js

```diff
@@ -108,7 +108,7 @@ class TagInput extends React.Component {
             onKeyPress={this.handleKeyPress.bind(this)}
             placeholder={this.props.intl.formatMessage(messages.search)}
           />
-         <Box ml="auto">
+        <Box ml="auto">
```

Comment:
I mean it needed one space more, not one less! Now it is two spaces behind! :joy:

Reply:
No prob @amoedoamorim hope it's alright now.